features/consent/src/main/res/layout/user_verification_fragment.xml

```diff
@@ -26,6 +26,24 @@
         android:layout_height="match_parent"
         tools:context=".ui.fragment.ConfirmPinFragment">
 
+        <TextView
+                android:id="@+id/lblBlockUser"
```

Comment:
I think top padding required bit more. can you pls check figma?

Reply:
its correct!